test/pupil-hpa/features/support/sql_db_helper.rb

```diff
@@ -463,4 +463,25 @@ def self.set_school_as_test_school(dfe_number)
     result = SQL_CLIENT.execute(sql)
     result.do
   end
+
+  def self.received_check(check_code)
+    sql = "SELECT * FROM [mtc_admin].[check] WHERE checkCode = '#{check_code}' and received=1"
```

Comment:
This is insecure.  Please use parameters.

Reply:
@jon-shipley As discussed, we will look at using parameters in a future PR

Reply:
is there not an API for this in the `SQL_CLIENT`? it should be really trivial to add...